spec.bs

```diff
@@ -129,22 +128,22 @@ spec: html; urlPrefix: https://www.rfc-editor.org/rfc/
   - <dfn for="epoch">taxonomy version</dfn>: a string.
   - <dfn for="epoch">model version</dfn>: a string.
   - <dfn for="epoch">config version</dfn>: a string.
-  - <dfn for="epoch">top 5 topics with caller origins</dfn>: a list of [=topic with caller origins=].
+  - <dfn for="epoch">top 5 topics with caller domains</dfn>: a list of [=topic with caller domains=].
   - <dfn for="epoch">time</dfn>: a {{DOMHighResTimeStamp}} (from Unix epoch).
 
-  A <dfn for="browsing topics types">topic with caller origins</dfn> is a struct with the following fields:
-  - <dfn for="topic with caller origins">topic id</dfn>: an integer.
-  - <dfn for="topic with caller origins">caller origins</dfn>: a set of [=origins=].
+  A <dfn for="browsing topics types">topic with caller domains</dfn> is a struct with the following fields:
+  - <dfn for="topic with caller domains">topic id</dfn>: an integer.
+  - <dfn for="topic with caller domains">caller domains</dfn>: a set of [=registrable domains=].
```

Comment:
The "type" is "domain", not "registrable domain". Here and in the rest of the section.

You could add a clarifying note at the end, saying something like

> All [=domains=] used in this API will be result of obtaining the [=registrable domain=] from some [=host=].